docs/docs/conditional-rendering.md

```diff
@@ -2,8 +2,6 @@
 id: conditional-rendering
 title: Conditional Rendering
 permalink: docs/conditional-rendering.html
-prev: handling-events.html
```

Comment:
My thought was that they are no longer part of an ordered navigation - each one is now either a stand-alone package, or deprecated. They soon won't live in the main React documentation. The 'next' and 'prev' links only make sense when they are all part of the set of 'addons', but the concept of the set itself is deprecated.